test/zdtm/static/bpf_array.c

```diff
@@ -70,6 +80,7 @@ int main(int argc, char **argv)
 		.map_flags = BPF_F_NUMA_NODE,
 	};
 	DECLARE_LIBBPF_OPTS(bpf_map_batch_opts, opts, .elem_flags = 0, .flags = 0, );
```

Comment:
Again, not related to this commit, but no need to explicitly initialize to `0` here.